tests/test_pickle_serialization.py

```diff
@@ -18,7 +18,7 @@
 from nose_parameterized import parameterized
 from unittest import TestCase
 
-from .serialization_cases import (
+from tests.serialization_cases import (
```

Comment:
I was having problems in PyCharm with relative imports but I can revert this if needs be.